src/Highlighter.js

```diff
@@ -34,6 +35,7 @@ export default function Highlighter ({
   autoEscape,
   caseSensitive = false,
   className,
+  customFindChunks,
```

Comment:
The docs mention `customFindChunks` but you use `findChunks` (in the `propTypes`) in the actual component. I actually prefer the `findChunks` naming, but I don't feel strongly.

Just update the `propTypes` to match.

Reply:
I named it `findChunks` first. But then further down in core's `utils.js` I passed it as default argument like `findChunks = findChunks` (first findChunks (left) being the argument being passed in like a custom findChunks, and the second argument being the local default `findChunks` being defined further down. The sense of this was kind of like: If a custom `findChunks` is then use the one defined further down. 

This did not work. Because both `findChunks` (left and right) are evaluated to the same (at least when looking at what babel did out if it, not sure how JS standards would define things here). 

So the easiest was just to name it `customFindChunks` all the way down/up.

I am happy if you have a better idea here? Otherwise I'd just keep it simple and leave it `customFindChunks`.

Reply:
I was just pointing out that you use two names in this component (one for prop types definition and another for the actual prop) and they need to match. 😄 

If you wanted to use the param name `findChunks` in the core lib, one way to do this would be to just rename the `const findChunks` further down to something that wouldn't class, like `const defaultFindChunks`. 😄 I kind of prefer this approach to naming.